site/book/04-using-functions/01-declarative-function-execution.md

```diff
@@ -126,6 +126,9 @@ The general way to provide a `functionConfig` of arbitrary kind (core or custom
 resources), is to declare the resource in a separate file in the same directory
 as the `Kptfile` and refer to it using the `configPath` field.
 
+You may use the short-hand form to specify value of `image` field.
```

Comment:
This doesn't belong in this section which is about `configPath`. Suggestion:

Adds this section to line 112:

## Specifying `image`

The `image` field specifies the container image for the function. You can specify an image form any container registry. If the registry is omitted, the default container registry for functions catalog (`gcr.io/kpt-fn`) is prepended automatically. For example, `set-labels:v0.1` is automatically expanded to `gcr.io/kpt-fn/set-labels:v0.1`.

Reply:
Updated. Thank you.